envoy/api/v2/route/route.proto

```diff
@@ -803,7 +814,36 @@ message HeaderMatcher {
   // * The regex *\d{3}* matches the value *123*
   // * The regex *\d{3}* does not match the value *1234*
   // * The regex *\d{3}* does not match the value *123.456*
-  google.protobuf.BoolValue regex = 3;
+  google.protobuf.BoolValue regex = 3 [deprecated = true];
```

Comment:
We shouldn't deprecate until the new one is ready yet :)

Reply:
will remove the tag.